protostar.py

```diff
@@ -53,12 +63,47 @@ def directory(directory_path: str) -> Path:
 )
 cmd_test_parser.add_argument(
     "--cairo-path-recursive",
-    "-cpr",
     nargs="+",
     default=[],
     type=directory,
     help="Paths which will be searched recursively in test contracts compilation",
     required=False,
 )
 
+
+cmd_compile_parser = root_subparsers.add_parser("compile")
+cmd_compile_parser.add_argument(
+    "input-files", type=cairo_file, nargs="+", help="Input files"
+)
+cmd_compile_parser.add_argument(
+    "--output",
+    type=argparse.FileType("w", encoding="utf-8"),
```

Comment:
This is nice, but I am little bit worried that you have to remember to close it everywhere, do you have any idea how to prevent forgetting it?

Reply:
That's not opening the file handle itself, it should be done by `argparse` module itself automatically